packages/bun-uws/src/HttpParser.h

```diff
@@ -263,23 +262,19 @@ namespace uWS
                    hasMore(x, 'z');
         }
 
-        static inline void *consumeFieldName(char *p)
-        {
-            for (; true; p += 8)
-            {
-                uint64_t word;
-                memcpy(&word, p, sizeof(uint64_t));
-                if (notFieldNameWord(word))
-                {
-                    while (isFieldNameByte(*(unsigned char *)p))
-                    {
-                        *(p++) |= 0x20;
-                    }
-                    return (void *)p;
+        static inline void *consumeFieldName(char *p) {
+        //for (; true; p += 8) {
+            //uint64_t word;
+            //memcpy(&word, p, sizeof(uint64_t));
+            //if (notFieldNameWord(word)) {
+                while (isFieldNameByte(*(unsigned char *)p)) {
+                    *(p++) |= 0x20;
                 }
-                word |= 0x2020202020202020ull;
-                memcpy(p, &word, sizeof(uint64_t));
-            }
+                return (void *)p;
+            //}
+            //word |= 0x2020202020202020ull;
+            //memcpy(p, &word, sizeof(uint64_t));
```

Comment:
Is the commented out code debug or likely to be reverted?

Reply:
unlikely to be reverted but this is the official committed version:
https://github.com/uNetworking/uWebSockets/commit/570fe762c82383b53765aeef3cafb65863c3abbc

Reply:
Ah I see. We are literally cherry-picking their patch.